src/benchmarks/micro/libraries/Microsoft.Extensions.DependencyInjection/ServiceProviderEngineBenchmark.cs

```diff
@@ -5,19 +5,21 @@
 using System;
 using System.Runtime.CompilerServices;
 using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
 
 namespace Microsoft.Extensions.DependencyInjection.Performance
 {
+    [BenchmarkCategory(Categories.Libraries)]
     public class ServiceProviderEngineBenchmark
     {
-        internal ServiceProviderMode ServiceProviderMode { get; private set; }
-
-        [Params("Expressions", "Dynamic", "Runtime", "ILEmit")]
-        public string Mode {
-            set {
-                ServiceProviderMode = (ServiceProviderMode)Enum.Parse(typeof(ServiceProviderMode), value);
-            }
-        }
+        // internal ServiceProviderMode ServiceProviderMode { get; private set; }
```

Comment:
Hey @halter73 @davidfowl do you know what the history here is? Specifically, are these benchmarks still relevant? Given I don't have much context here, I'm inclined to keep the benchmarks for each of the DI modes, but if these are no longer useful in any way, we can remove them.

Reply:
>  Given I don't have much context here, I'm inclined to keep the benchmarks for each of the DI modes, but if these are no longer useful in any way, we can remove them.

I'm inclined to keep them too. Even if we aren't doing active perf work on DI now, it could be useful later. Is there some reason we cannot make Microsoft.Extensions.DependencyInjection's internals visable to this project?

Reply:
The problem is that the benchmarks app isn't strong named so IVT doesn't work. But I think what we should do here is turn strong name on, instead of removing the benchmarks. @maryamariyan is looking into do that now.

Reply:
moved over the PR to https://github.com/dotnet/performance/pull/1362/commits/48ba3c46ecd4e62b5ab27fd012fdd0880a0797e0#r441228858